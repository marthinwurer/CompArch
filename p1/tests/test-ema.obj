100 1 00001
101 1 00005
102 1 1ffff
103 1 1fffb
150 1 00ace
200 1 10100
201 1 14150
202 1 14150
203 1 10101
204 1 14150
205 1 14150
206 1 10102
207 1 14150
208 1 14150
209 1 10103
20a 1 14150
20b 1 14150
20c 1 00000
200
