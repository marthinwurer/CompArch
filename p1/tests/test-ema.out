CPU "ARCH" Simulator, 2.5a(Jan 22 2015)
-----------------------------------------

Memory sets starting address to 200
200:  10100  LDA   100  A[00000]  B[00000]  XR[000]
201:  14150  EMA   150  A[00001]  B[00000]  XR[000]
202:  14150  EMA   150  A[00ace]  B[00000]  XR[000]
203:  10101  LDA   101  A[00001]  B[00000]  XR[000]
204:  14150  EMA   150  A[00005]  B[00000]  XR[000]
205:  14150  EMA   150  A[00ace]  B[00000]  XR[000]
206:  10102  LDA   102  A[00005]  B[00000]  XR[000]
207:  14150  EMA   150  A[1ffff]  B[00000]  XR[000]
208:  14150  EMA   150  A[00ace]  B[00000]  XR[000]
209:  10103  LDA   103  A[1ffff]  B[00000]  XR[000]
20a:  14150  EMA   150  A[1fffb]  B[00000]  XR[000]
20b:  14150  EMA   150  A[00ace]  B[00000]  XR[000]
20c:  00000  HALT       A[1fffb]  B[00000]  XR[000]

Machine Halted - HALT instruction executed

Simulated time 84 cycles

LAST CPUObject DESTROYED; END OF SIMULATION
